src/utils/codec.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { base64url } from "rfc4648";
+import bs58 from "bs58";
+
+export const convertBase64urlToBase58 = (value: string): string => {
```

Comment:
_sigh_ at our inability as an ecosystem to standardize so this is a thing

Reply:
I looked for libraries that would handle this as well as other codec concerns, but it seemed that the typing for them all made them either browser specific / node specific so I figured this was the simpler way to approach it.